Formula/libfreenect.rb

```diff
@@ -13,13 +13,33 @@ class Libfreenect < Formula
     sha256 "f714532e1b21365063746846544a340dac70cf0c5cc877a207dd17284ee100b7" => :mavericks
   end
 
+  option :universal
+  option "with-opencv", "Build with OpenCV support"
+  option "with-opencv3", "Build with OpenCV 3 support"
+  option "with-python", "Build with Python support"
+  option "with-python3", "Build with Python 3 support"
```

Comment:
Please remove all of the `option` calls (including `:universal`, unless there's a really compelling use case?); the `depends_on... => :optional` syntax does this for you.